runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -566,7 +566,7 @@ def up(self, verbose: bool = True, force: bool = False):
             DenLauncher.up(cluster=self, verbose=verbose, force=force)
 
         elif self.launcher_type == LauncherType.LOCAL:
-            logger.info("Launching cluster locally")
```

Comment:
re-wording here since this is running in the launcher in local mode, so when streamed back would look a little odd

Reply:
Agreed that was not great before!